packages/api/src/interactors/RemoveUserFromOrganizationInteractor.ts

```diff
@@ -78,6 +80,16 @@ export class DeleteUserInteractor implements Interactor<MutationDeleteUserArgs,
 			return new FailedResponse(this.localization.t('mutation.deleteUser.fail'))
 		}
 
+		try {
+		} catch (error) {
+			return new FailedResponse(this.localization.t('mutation.deleteUser.fail'))
```

Comment:
Why did we decide to revert to using FailedResponse instead of using the Error responses that Apollo provides?